newrelic/hooks/datastore_redis.py

```diff
@@ -377,8 +364,11 @@
     "smembers",
     "smismember",
     "smove",
+    "sort_ro",
+    "sort",
     "spellcheck",
-    "spublish",
```

Comment:
Was this command supposed to be removed? I see the tests are failing because `spublish` is showing up as uninstrumented. I didn't see this in the diff for the original PR (unless I missed it)

Reply:
Yeah, it wasn't there in the original PR...it got added later on).  Might as well add it back in here.